### PR TITLE
Add new RepresentationHandler to support text column

### PR DESCRIPTION
**Pull request by binhmop**

### CODE REVIEW

File: spark/src/test/scala/io/ddf/spark/analytics/MLSupporterSuite.scala [hunk 1]
```diff
@@ -3,11 +3,12 @@ package io.ddf.spark.analytics
 import io.ddf.DDF
 import io.ddf.ml.IModel
 import io.ddf.spark.{ATestSuite, SparkDDF}
+import org.apache.spark.rdd._
 
 /**
   */
 class MLSupporterSuite extends ATestSuite {
```
Comment (hai-adatao) [inline spark/src/test/scala/io/ddf/spark/analytics/MLSupporterSuite.scala]: Another empty test class ...
I would vote for remove it or write a test for it

Comment (hai-adatao) [inline spark/src/test/scala/io/ddf/spark/analytics/MLSupporterSuite.scala]: Another empty test class ...
I would vote for remove it or write a test for it

File: spark/src/main/scala/io/ddf/spark/content/RDDRow2ArrayString.scala [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package io.ddf.spark.content
+
+import io.ddf.DDF
+import io.ddf.content.{ConvertFunction, Representation}
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.Row
+
+class RDDRow2ArrayString (@transient ddf: DDF) extends ConvertFunction(ddf) {
```
Comment (hai-adatao) [inline spark/src/main/scala/io/ddf/spark/content/RDDRow2ArrayString.scala]: Is this testable? Can we have a test for this?

**Comment by hai-adatao** [comment]
@steveyan777 @sameerp @mbbui

File: spark/src/test/scala/io/ddf/spark/analytics/MLSupporterSuite.scala (hunk 1 above)
Comment (binhmop) [inline spark/src/test/scala/io/ddf/spark/analytics/MLSupporterSuite.scala]: It seems that it was moved to pAnalytics/ddf_adatao/src/test/scala/com/adatao/ddf/spark/analytics/MLSupporterSuite.scala
so I put the unit test there

File: spark/src/main/scala/io/ddf/spark/content/RDDRow2ArrayString.scala [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+  override def apply(representation: Representation): Representation = {
+    representation.getValue match {
+      case rdd: RDD[Row] =>
+        val rddArrString = rdd.map {
+          row => {
+            row.getString(0).split("[ ,]")
+          }
+        }.filter(row => row != null)
```
Comment (Huandao0812) [inline spark/src/main/scala/io/ddf/spark/content/RDDRow2ArrayString.scala]: if the row is null, there'll be NPE at line 15, so properly won't need to filter null at this line

Comment (binhmop) [inline spark/src/main/scala/io/ddf/spark/content/RDDRow2ArrayString.scala]: good catch, removing...

**Pull request merged**